cds/modules/deposit/static/js/cds_deposit/avc/components/cdsDeposit.js

```diff
@@ -93,6 +94,46 @@ function cdsDepositCtrl(
       }
     };
 
+    this.inheritMetadata = function() {
+      var record = that.record;
+      var master = that.cdsDepositsCtrl.master.metadata;
+      var accessElement = function(obj, elem, value) {
+        // Find an element inside an object given a path of properties
+        // If value is given, set element to value
+        var lastPart, parentObj = obj;
+        angular.forEach(elem.split('.'), function(part) {
```

Comment:
I think we can use ``ObjectPath`` https://github.com/mike-marcacci/objectpath here :)